webviz_subsurface/plugins/__init__.py

```diff
@@ -58,3 +58,4 @@
 from ._well_log_viewer import WellLogViewer
 from ._assisted_history_matching_analysis import AssistedHistoryMatchingAnalysis
 from ._well_completions import WellCompletions
+from ._group_tree import GroupTree
```

Comment:
(unsorted list of imports. fancy `isort`?)

Reply:
hva do you think @anders-kiaer ?

Reply:
👍🏻 Leave it as a new issue atm, we can look at isort (+"fix" sorting in other modules as well) in a separate PR

Reply:
https://github.com/equinor/webviz-subsurface/issues/789